### ISSUE TITLE
Can it be used for shiny?

### ISSUE THREAD

**Issue by Liripo** [JosiahParry/valve#11]
I would like to ask you if it is possible to use it for shiny?

**Comment by JosiahParry** [comment]
No, it is not posisble to use it for shiny right now. I am unsure how I can accomplish preserving state at the moment. If you have the wherewithal and knowledge of Rust to give it a shot, it would be _greatly_ appreceiated.

**Comment by JosiahParry** [comment]
I attempted this with limited success in this branch https://github.com/JosiahParry/valve/tree/shiny/inst

**Comment by Liripo** [comment]
Thank you for your reply.

**Comment by JosiahParry** [comment]
Reopening this for myself. To support shiny rather than act as a reverse proxy we might want to just forward the connection. Figuring out timeouts might be a bit trickier though...

**Comment by JosiahParry** [comment]
No, it cannot be used for shiny. If you would like to support shiny i recommend checking out faucet! This is specifically for autoscaling plumber APIs.
